python/pyspark/ml/tuning.py

```diff
@@ -445,7 +447,9 @@ def copy(self, extra=None):
         """
         if extra is None:
             extra = dict()
-        return TrainValidationSplitModel(self.bestModel.copy(extra))
+        bestModel = self.bestModel.copy(extra)
+        validationMetrics = list(self.validationMetrics)
```

Comment:
So this is a shallow copy rather than a deep copy - its probably OK since we aren't explicit in the docstring but maybe we should be explicit in the docstring that we make a shallow copy of the validation metrics since we are explicit about the other parts of the copy in the docstring?